3.0.0-RC4/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Gatling is a highly capable load testing tool.
+#
+# Documentation: https://gatling.io/docs/2.3/
+# Cheat sheet: http://gatling.io/#/cheat-sheet/2.3
+
+FROM openjdk:8-jdk-alpine
+
+MAINTAINER Denis Vazhenin <denis.vazhenin@me.com>
+
+# working directory for gatling
+WORKDIR /opt
+
+# gating version
+ENV GATLING_VERSION 3.0.0-RC4
+
+# add compatibility libraries for librcrypto.so
+RUN apk add libc6-compat
+
+# create directory for gatling install
+RUN mkdir -p gatling
+
+# install gatling
+RUN apk add --update wget bash && \
```

Comment:
I think it would be better to move the installation of the `libgc-compact` dependency here